docs/source/markdown/podman-create.1.md

```diff
@@ -989,6 +999,14 @@ can override the working directory by using the **-w** option.
 $ podman create alpine ls
 ```
 
+### Run a container with just the SETUID and SETGID Capabilities
+
+
+```
+
+$ podman run -d --label=io.containers.capabilities=setuid,setgid fedora /usr/bin/server
```

Comment:
Why do I do this instead of `--cap-add` and `--cap-drop`? I don't know if this makes much sense outside of loading from images.

Reply:
The idea is to get this data into the container image.  And maybe I should add the data to Docker build.
The real goal is to get people to add 
LABEL io.containers.capabilities=setuid,setgid 
In their Containerfile/Dockerfile.
Then users of their container image will get this new more secure behaviour.